src/cloud-api-adaptor/podvm/Dockerfile.podvm_builder.rhel

```diff
@@ -6,9 +6,12 @@
 # Creates a builder container image that should be used to build the Pod VM
 # disk inside a container.
 #
-FROM registry.access.redhat.com/ubi9/ubi:9.2
+FROM registry.access.redhat.com/ubi9/ubi:9.4
 
 ARG ARCH
+# PROTOC_ARCH="x86_64" | "s390x_64"
+ARG PROTOC_ARCH="x86_64"
```

Comment:
for s390x we need PROTOC_ARCH as s390_64 else it will eventually fail with exec format error.

Reply:
@snir911 should the comment be changed to s390_64 based on the above comment ?

Reply:
@snir911 this comment `# PROTOC_ARCH="x86_64" | "s390x_64"` can be removed.
I see in Makefile PROTOC_ARCH is explicitly set to s390x
```
ifeq ($(ARCH),s390x)
        docker buildx build \
                -t $(BUILDER) \
                --build-arg ARCH=s390x \
                --build-arg PROTOC_ARCH=s390x \
                --build-arg YQ_ARCH=s390x \
                --build-arg YQ_CHECKSUM=sha256:4e6324d08630e7df733894a11830412a43703682d65a76f1fc925aac08268a45 \
                --load \
                -f ../podvm/Dockerfile.podvm_builder.fedora ../.
else
```

Reply:
@bpradipt AFAIU I'd leave it there for users who use the Dockerfiles directly.
BTW In the code snippet you showed i think we can convert avoid conditional the ARCH similarly to the current podvm/Makefile, better to be done in different PR i guess though..

Reply:
nit: remove the `# PROTOC_ARCH="x86_64" | "s390x_64"` string from this commit as it's updated to the right one in the next commit `# PROTOC_ARCH="x86_64" | "s390_64"`.

Reply:
@snir911